engine/installation/linux/oracle.md

```diff
@@ -16,27 +16,36 @@ To get started with Docker on Oracle Linux, make sure you
 
 To install Docker, you need the 64-bit version of Oracle Linux 6 or 7.
 
-To use `btrfs`, you need to install the Unbreakable Enterprise Kernel (UEK)
+Oracle recommends using the Unbreakable Enterprise Kernel (UEK)
 version 4.1.12 or higher. For Oracle Linux 6, you need to enable extra repositories
 to install UEK4. See
 [Obtaining and installing the UEK packages](https://docs.oracle.com/cd/E37670_01/E37355/html/ol_obtain_uek.html){: target="_blank" class="_" }.
 
+The [OverlayFS2 storage driver](https://docs.docker.com/engine/userguide/storagedriver/overlayfs-driver/) is only supported
+when running the UEK4.
+
 ### Remove unofficial Docker packages
 
-Oracle's repositories contain an older version of Docker, with the package name
+Oracle's repositories may contain an older version of Docker, with the package name
 `docker` instead of `docker-engine`. If you installed this version of Docker,
 remove it using the following command:
 
 ```bash
 $ sudo yum -y remove docker
 ```
 
+Oracle's repositories also contain the Oracle-supported version of Docker which uses
```

Comment:
This seems superfluous, since the right thing will happen anyway: The latest version will always be chosen.